gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/main/java/io/gravitee/gateway/reactor/ReactorEvent.java

```diff
@@ -24,5 +24,4 @@ public enum ReactorEvent {
     UPDATE,
     UNDEPLOY,
     DEBUG,
-    SUBSCRIPTION,
```

Comment:
is it related to acceptor ?

Reply:
It's been merged from a previous PR but is not relevant.
And will not be relevant in the future with the introduction of subscription
So I choose to remove it